core/src/main/java/com/gcinterceptor/core/SheddingThreshold.java

```diff
@@ -36,6 +36,10 @@
 		threshold = new AtomicLong((long) (MIN_SHEDDING_THRESHOLD + (Math.random() * MIN_SHEDDING_THRESHOLD)));
 	}
 
+	int numbGCs() {
```

Comment:
Rename to numGCs (note there is a b in the middle of the method name)

Reply:
My bad, sorry. Consider ir fixed :)